qemu/tests/drive_mirror_cancel.py

```diff
@@ -24,7 +24,7 @@ def run_drive_mirror_cancel(test, params, env):
         bg = utils.InterruptedThread(mirror_test.cancel,)
         bg.start()
         job = mirror_test.get_status()
-        if job["type"] != "mirror":
+        if job.get("type", "0") != "mirror":
```

Comment:
@xutian , since there is nothing returned when execute query-block-job after job canceled, so "type" would be always not exist and therefore "raise error.TestFail("Job cancel immediacatly")" line would be never executed. I met this situation in my test, so adjust it like this.